libs/langchain/langchain/tools/vectorstore/tool.py

```diff
@@ -17,6 +17,8 @@ class BaseVectorStoreTool(BaseModel):
 
```

Comment:
I was just wondering why BaseVectorStoreTool is not derived from BaseTool? Maybe it is worth adding a comment about it.

Reply:
Good question, but I don't know the answer and I think it's out of scope for this PR since it's not for the GitHub tool. 
This file is only changed to pass through `kwargs` to the retriever. Thanks.